packages/beacon-node/src/chain/historicalState/diffLayers.ts

```diff
@@ -0,0 +1,104 @@
+import {Slot} from "@lodestar/types";
+import {SLOTS_PER_EPOCH} from "@lodestar/params";
+import {StateArchiveStrategy} from "./types.js";
+
+export const DEFAULT_DIFF_LAYERS = "4, 64, 256, 1024";
```

Comment:
need a comment and benchmark/evidence on how do we come up with this default value and what does this mean